src/System.Runtime.InteropServices/tests/System/Runtime/InteropServices/ComEventsHelperTests.cs

```diff
@@ -9,63 +9,63 @@ namespace System.Runtime.InteropServices.Tests
 #pragma warning disable 0618 // ComEventsHelper is marked as Obsolete.
     public class ComEventsHelperTests
     {
-        [Fact]
-        [PlatformSpecific(TestPlatforms.Windows)]
-        [SkipOnTargetFramework(TargetFrameworkMonikers.UapAot, "Throws PlatformNotSupportedException in UapAot")]
-        public void Combine_NullRcwWindows_ThrowsArgumentNullException()
-        {
-            AssertExtensions.Throws<ArgumentNullException>(null, () => ComEventsHelper.Combine(null, Guid.Empty, 1, null));
-        }
-
         [Fact]
         [SkipOnTargetFramework(~TargetFrameworkMonikers.UapAot, "Throws PlatformNotSupportedException in UapAot")]
-        public void Combine_NullRcwUapAot_PlatformNotSupportedException()
+        public void Combine_UapAot_PlatformNotSupportedException()
         {
             Assert.Throws<PlatformNotSupportedException>(() => ComEventsHelper.Combine(null, Guid.Empty, 1, null));
         }
         
         [Fact]
         [PlatformSpecific(TestPlatforms.AnyUnix)]
-        public void Combine_NullRcwUnix_ThrowsArgumentNullException()
+        public void Combine_Unix_ThrowsArgumentNullException()
         {
             Assert.Throws<PlatformNotSupportedException>(() => ComEventsHelper.Combine(null, Guid.Empty, 1, null));
         }
 
         [Fact]
-        [SkipOnTargetFramework(TargetFrameworkMonikers.NetFramework, "ComEventsHelper.Combine is not supported in .NET Core.")]
-        [ActiveIssue(31214)]
-        public void Combine_NonNullRcw_ThrowsPlatformNotSupportedException()
+        [PlatformSpecific(TestPlatforms.Windows)]
+        [SkipOnTargetFramework(TargetFrameworkMonikers.UapAot, "Throws PlatformNotSupportedException in UapAot")]
+        public void Combine_NullRcw_ThrowsArgumentNullException()
         {
-            Assert.Throws<PlatformNotSupportedException>(() => ComEventsHelper.Combine(1, Guid.Empty, 1, null));
+            AssertExtensions.Throws<ArgumentNullException>(null, () => ComEventsHelper.Combine(null, Guid.Empty, 1, null));
         }
 
         [Fact]
         [PlatformSpecific(TestPlatforms.Windows)]
-        [SkipOnTargetFramework(TargetFrameworkMonikers.UapAot, "Throws PlatformNotSupportedException in UapAot")]
-        public void Remove_NullRcwWindows_ThrowsArgumentNullException()
+        [SkipOnTargetFramework(TargetFrameworkMonikers.NetFramework, "ComEventsHelper.Combine is not supported in .NET Core.")]
```

Comment:
Sorry, does this say this attribute now say skip on `NetFramework`, but the comment says the function isn't supported on .NET Core?